schemas/client.ts

```diff
@@ -0,0 +1,50 @@
+import {defineField, defineType} from 'sanity'
+
+export default defineType({
+  name: 'client',
+  title: 'Client',
+  type: 'document',
+  fields: [
+    defineField({
+      name: 'name',
+      title: 'Name',
+      type: 'string',
+    }),
```

Comment:
In the file above, you use empty lines between define fields block, but not here and below

Reply:
Fixed ✅